financial-connections/src/main/java/com/stripe/android/financialconnections/utils/UriUtils.kt

```diff
@@ -23,6 +23,15 @@ internal class UriUtils @Inject constructor(
             uri1.path.equals(uri2.path)
     }
 
+    fun addQueryParameterToUri(
+        uri: String,
+        queryParam: Pair<String, String>
+    ): String? {
+        val uriBuilder = uri.toUriOrNull()?.buildUpon()
+        uriBuilder?.appendQueryParameter(queryParam.first, queryParam.second)
```

Comment:
In my experience, this creates an invalid URL because we use URI fragments. Let’s just do `authUrl + "&return_payment_method=true"`.